src/main/external-resources/renderers/Philips-PFL.conf

```diff
@@ -147,7 +142,7 @@ RendererIcon = philips-pfl.png
 #
 
 UserAgentSearch = PhilipsIntelSDK
-UpnpDetailsSearch = Philips , NMR
+UpnpDetailsSearch = Philips TV
```

Comment:
@SubJunk Do you mean that from the referenced log ?
> DEBUG 2017-01-22 15:15:10.540 [cling-5] net.pms.network.UPNPHelper New renderer found: "Philips TV" with dlna details: {friendlyName=Philips TV, address=192.168.0.10, udn=uuid:F00DBABE-SA5E-BABA-DADA188ED5035A8E, manufacturer=Philips, modelName=NMR, modelNumber=1.0, modelDescription=UPnP Media Renderer 1.0, manufacturerURL=http://www.philips.com, modelURL=http://www.philips.com/}

Or the same from UMS log from 2014:
http://pastebin.com/J1rXBVEg

That's said i need to add User-Agent string as well, as i suspect strongly user with badly configurated network to acceed mainly to this string more than the UPnP one.